cmd/lakectl/cmd/docs.go

```diff
@@ -59,7 +59,7 @@ lakectl config
 ` + "```" + `
 
 This will setup a ` + "`$HOME/.lakectl.yaml`" + ` file with the credentials and API endpoint you've supplied.
-When setting up a new installation and creating initial credentials (see [Quick start](../quickstart/index.md)), the UI
+When setting up a new installation and creating initial credentials (see [Quick start](/quickstart/index.md)), the UI
```

Comment:
If I remember the relative path was to support reference the document relative to the current one - if we use abs path we need to verify it will work on versioned document.
When we release the same site is generated under the version vX.X.X/... - need to verify we will not reference the wrong document.

Reply:
I'm glad you reviewed this :) 
I built it locally with versions and confirmed that the absolute pathing will stop the references within versioned docs working. 
I've reverted the change and made sure that all links are now relative.